src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/dates/ts.ui.TimeSpirit.js

```diff
@@ -1,21 +1,43 @@
 /**
  * Spirit of the time.
  * @extends {ts.ui.Spirit}
+ * @using {Moment.js} moment
  * @using {gui.Combo.chained}
  */
-ts.ui.TimeSpirit = (function using(chained) {
+ts.ui.TimeSpirit = (function using(moment, chained) {
 	var mytick = 'mytick';
 	var time = 1000;
 	var issame = false;
 	var firsttime = null;
 
+	/**
+	 * There's two problems. First, our language code `no` for Nynorsk is not 
+	 * correctly declared in V4 as `nn-NO` according to the ISO standard and 
```

Comment:
small typo in `nb-NO`

Reply:
No, `nn-NO` is the certified ISO language code that we wrongly tried to target with `no`.